plyer/platforms/win/sysinfo.py

```diff
@@ -0,0 +1,64 @@
+import platform
+import os
+import re
+from subprocess import Popen, PIPE
+from plyer.facades import Sysinfo
+from plyer.platforms.win.libs import screenmetric
+
+
+class WindowsSysinfo(Sysinfo):
+
+    values = {}
+
+    def __init__(self, **kwargs):
+        self.SysInfo()
+
+    def SysInfo(self):
+
+        cache = os.popen2("SYSTEMINFO")
```

Comment:
No, I would suggest deferring this call. Make it check if `self.values` is set, and return immediately if it is, then you can call this in every other method before accessing `self.values`. Also, please remember to follow naming conventions, and also this is a private method, so calling it something like `_get_sysinfo` or `_ensure_sysinfo` would be better.